src/NzbDrone.Core.Test/Datastore/Migration/075_force_lib_updateFixture.cs

```diff
@@ -57,6 +57,14 @@ public void should_reset_series_last_sync_time()
         {
             WithTestDb(c =>
             {
+                c.Insert.IntoTable("Profiles").Row(new
```

Comment:
Why do these old tests need to be updated? Changing a migration test is a bit of a concern because we need to be very sure nothing breaks for people running old DB versions.